components/director/internal/domain/application/service.go

```diff
@@ -265,6 +266,8 @@ func (s *service) Create(ctx context.Context, in model.ApplicationRegisterInput)
 	if in.IntegrationSystemID != nil {
 		in.Labels[intSysKey] = *in.IntegrationSystemID
 	}
+	//This is workaround for connectivity adapter https://github.com/kyma-incubator/compass/tree/master/docs/investigations/rest-adapters
```

Comment:
I don't know if this comment is necessary. I don't know if this is a workaround either 😄 We want to filter via labels, right? So it makes sense to put all properties there

Reply:
Removed : )